pkg/controller/migmigration/task.go

```diff
@@ -608,8 +656,8 @@ func (t *Task) next() {
 	t.Phase = Completed
 }
 
-// Phase failed.
-func (t *Task) failed(nextPhase string, reasons []string) {
+// Phase fail.
+func (t *Task) fail(nextPhase string, reasons []string) {
```

Comment:
Why was this renamed?

Reply:
In correlation to `t.next()` This phase is never **next**, it is just an order to do something, either `fail` or go `next`.

Also, it correlates with `t.failed() bool` check, saying if migration failed, which is used at https://github.com/konveyor/mig-controller/blob/c2752422b1e329e68e12e9e982b4579f54838d3a/pkg/controller/migmigration/task.go#L604 and there is no way around.